src/content/docs/sensitive-info/quick-start/snippets/bun/Step3.ts

```diff
@@ -0,0 +1,39 @@
+import arcjet, { sensitiveInfo } from "@arcjet/bun";
+import { env } from "bun";
+
+// This function is called by the `sensitiveInfo` rule to perform custom detection on strings.
+function detectDash(tokens: string[]): Array<"CONTAINS_DASH" | undefined> {
+  return tokens.map((token) => {
+    if (token.includes("-")) {
+      return "CONTAINS_DASH";
+    }
+  });
+}
+
+const aj = arcjet({
+  key: env.ARCJET_KEY!, // Get your site key from https://app.arcjet.com
+  rules: [
+    // This allows all sensitive entities other than email addresses and those containing a dash character.
+    sensitiveInfo({
+      // allow: ["EMAIL"], Will block all sensitive information types other than email.
+      deny: ["EMAIL", "CONTAINS_DASH"], // Will block email and any custom detected values that "contain dash"
+      mode: "LIVE", // Will block requests, use "DRY_RUN" to log only
+      detect: detectDash,
+      contextWindowSize: 2, // Two tokens will be provided to the custom detect function at a time.
```

Comment:
This isn't explained anywhere - can you add a section in the reference to describe what it does and when you'd use it?

Reply:
This = `contextWindowSize`

Reply:
Yeah I'll add one in.

Reply:
I've added this to the custom detect function section in the bun reference docs. Once you're happy with it I'll copy that section across to the other adapters.